### ISSUE TITLE
Level 17 has wrong Example code

### ISSUE THREAD

**Issue by zaelcovsky** [kiksen1987/xpath-diner#14]
Code example:
`
A//B will select all B that are a direct children A
`
should be with one "/":
`A/B will select all B that are a direct children A`
because it is **direct children**

**Comment by ardhrubo** [comment]
Yes, this issue is still there and hasn't been solved. Look at the example.
![image](https://github.com/user-attachments/assets/afc85e87-3524-488c-8ea4-747760bb498d)

**Comment by roslovets** [comment]
Fixed in fork https://github.com/Inzhenerka/xpath-diner
Thanks

**Comment by zaelcovsky** [comment]
Great news, mate
